jaeger-zipkin/src/main/java/com/uber/jaeger/senders/zipkin/ZipkinSender.java

```diff
@@ -125,11 +124,6 @@ public int append(Span span) throws SenderException {
     return n;
```

Comment:
I think I finally figured out your intent.. sorry you are saying remove "queuedSizeInBytes". doing that now!